File: QViz.NET/Periscope.QViz/JSON/ToolJira.cs
namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Tool Configuration Data for JIRA Software
	/// </summary>
	public class ToolJira
	{
		/// <summary>
		/// Name of the Tool Configuration (JIRA)
		/// </summary>
		public string toolName;

		/// <summary>
		/// URL of the JIRA REST v2 API Instance
		/// </summary>
		public string url;

		/// <summary>
		/// Unique Name of the JIRA User
		/// </summary>
		public string userName;

		/// <summary>
		/// JIRA REST v2 API Access Token of the User
		/// </summary>
		public string password;

		/// <summary>
		/// Unique Key of the JIRA Project
		/// </summary>
		public string jiraProjectKey;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public ToolJira()
		{
			this.toolName = "";
			this.url = "";
			this.userName = "";
			this.password = "";
			this.jiraProjectKey = "";
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/ToolJira.java
package com.periscope.qviz.json;

/**
 * JSON Model for Tool Configuration Data for JIRA Software
 */
public class ToolJira {

	/**
	 * Name of the Tool Configuration (JIRA)
	 */
	public String toolName;

	/**
	 * URL of the JIRA REST v2 API Instance
	 */
	public String url;

	/**
	 * Unique Name of the JIRA User
	 */
	public String userName;

	/**
	 * JIRA REST v2 API Access Token of the User
	 */
	public String password;

	/**
	 * Unique Key of the JIRA Project
	 */
	public String jiraProjectKey;

	/**
	 * Default Constructor
	 */
	public ToolJira() {
		this.toolName = "";
		this.url = "";
		this.userName = "";
		this.password = "";
		this.jiraProjectKey = "";
	}

}
